lib/charms/layer/jenkins/api.py

```diff
@@ -69,6 +71,18 @@ def update_password(self, username, password):
         client.run_script(UPDATE_PASSWORD_SCRIPT.format(
             username=username, password=password))
 
+    def get_plugin_version(self, plugin):
+        """Get the installed version of a given plugin
+
+        If the plugin is not installed returns False
+        """
+        client = self._make_client()
+        script = "println(Jenkins.instance.updateCenter.getPlugin(\"{}\")?.version)".format(plugin)
```

Comment:
It is common to use single quotes for the string to avoid needing to quote the double quote marks, and vice versa.

Reply:
Addressed in 2e5d3b6b01ce6008570eff7efb8690447b0ee9ee